remme/settings/__init__.py

```diff
@@ -7,7 +7,9 @@
 SETTINGS_PUB_KEY_ENCRYPTION = 'remme.settings.pub_key_encryption'
 SETTINGS_KEY_GENESIS_OWNERS = 'remme.settings.genesis_owners'
 SETTINGS_SWAP_COMMISSION = 'remme.settings.swap_comission'
-SETTINGS_STORAGE_PUB_KEY = 'remme.settings.storage_pub_key'
+
+# Backward compatibility to use more obvious variable name
+SETTINGS_KEY_TO_GET_STORAGE_PUBLIC_KEY_BY = SETTINGS_STORAGE_PUB_KEY = 'remme.settings.storage_pub_key'
```

Comment:
It became even worse :) What's so wrong with the old one?

Reply:
@eugene-babichenko, `SETTINGS_STORAGE_PUB_KEY` says it is real public key, not key to get public key by.